### PR TITLE
Added total task cost (GNT) and fee (ETH) to comp.tasks

### LINKED ISSUE #2895: Additional parameters to "comp.tasks" RPC
To implement the prototype shown on the image below,  we need to add some additional parameters to the "com.tasks" RPC. 

- [ ] Final status date/timestamp, to show user when exactly the last action happened
- [x] Locked GNT & ETH amount for the specific task

image

### PR DESCRIPTION
**Pull request by Wiezzel**
Modified `comp.task` and `comp.tasks` to include two new fields:
* `cost` - total cost of the task (GNT)
* `fee` - total transaction fee for the task (ETH)

**Only confirmed transactions are counted.**

Closes #2895

### CODE REVIEW

**Comment by Wiezzel** [comment]
@mdtanrikulu Please check if this works when you're done with the front-end part.

**Review by maaktweluit — APPROVED** [review]
LGTM!

ihavesomesortofidea

File: tests/factories/model.py [hunk 1]
```diff
@@ -1,13 +1,32 @@
...
+    node_info = SubFactory(p2p.Node)
+    fee = Faker('pyint')
+
+
+class Payment(Factory):
+    class Meta:
+        model = model.Payment
+
+    subtask = Faker('uuid4')
+    payee = Faker('binary', length=64)
```
Comment (Krigpl) [inline tests/factories/model.py]: `length=20` - it's an Ethereum address

**Review by Krigpl — APPROVED** [review]

File: tests/factories/model.py (hunk 1 above)
Comment (Wiezzel) [reply tests/factories/model.py]: Fixed.

**Comment by mdtanrikulu** [comment]
@Wiezzel this need to be implemented as well; 

- [x] Final status date/timestamp, to show user when exactly the last action happened.

**Comment by Wiezzel** [comment]
@mdtanrikulu Done. Added `last_updated` field which contains the time of last status change. It is a timestamp like `time_started`.

**Review by mdtanrikulu — APPROVED** [review]

**Pull request merged**